pkg/core/blockchain.go

```diff
@@ -1132,6 +1207,152 @@ func (bc *Blockchain) GetScriptHashesForVerifyingClaim(t *transaction.Transactio
 	return nil, fmt.Errorf("no hashes found")
 }
 
+//GetStandByValidators returns validators from the configuration.
+func (bc *Blockchain) GetStandByValidators() (keys.PublicKeys, error) {
+	return getValidators(bc.config)
+}
+
+// GetValidators returns validators.
+// Golang implementation of GetValidators method in C# (https://github.com/neo-project/neo/blob/c64748ecbac3baeb8045b16af0d518398a6ced24/neo/Persistence/Snapshot.cs#L182)
+func (bc *Blockchain) GetValidators() ([]*keys.PublicKey, error) {
+	tx := &transaction.Transaction{}
+	return bc.GetValidatorsBy([]*transaction.Transaction{tx})
```

Comment:
In C# implementation empty collection is provided, not a single-element array.
And if so, may be it is better to merge `GetValidators()` and `GetValidatorsBy([]tx)` into `GetValidators(...tx)` ?

Reply:
done! Thank you!